indigo_app/views/tasks.py

```diff
@@ -136,6 +136,13 @@ def get_context_data(self, **kwargs):
 
         context['possible_workflows'] = Workflow.objects.unclosed().filter(country=task.country, locality=task.locality).all()
 
+        # TODO: other tasks on this and related works (if the task has a work, otherwise none)
```

Comment:
I think safest to be any task in this place.

Reply:
I've updated this to exclude closed tasks; we might need to filter it further in future so that not too many tasks are loaded.